frontend/js/src/playlists/PlaylistMenu.tsx

```diff
@@ -133,17 +141,8 @@ function PlaylistMenu({
       { toastId: "export-playlist" }
     );
   };
-  const handlePlaylistExport = async (
-    handler: (
-      playlistId: string,
-      playlistTitle: string,
-      auth_token: string
```

Comment:
I realized we don't need to pass those arguments to the handler functions, since they are defined in the main scope of the component function and we have access to those values from the get go.